src/stringProcessing/specialCharacterMappings.js

```diff
@@ -0,0 +1,176 @@
+const filter = require( "lodash/filter" );
+const includes = require( "lodash/includes" );
+const memoize = require( "lodash/memoize" );
+
+/**
+ * Gets indices of a specific character in the input text.
+ *
+ * @param {string} text The original text, for which the indices of specific characters have to be determined.
+ * @param {string} characterToFind The character that needs to be found in the text.
+ *
+ * @returns {Array} indices The array of indices in the text at which the characterToFind occurs.
+ */
+function getIndicesOfCharacter( text, characterToFind ) {
+	let indices = [];
```

Comment:
You can use `const` here because you push things onto the array. You do not re-assign it to another array.